zookeeper-server/src/test/java/org/apache/zookeeper/test/InvalidSnapshotTest.java

```diff
@@ -46,12 +45,13 @@
     /**
      * Verify the LogFormatter by running it on a known file.
      */
+    @SuppressWarnings("deprecation")
     @Test
     public void testLogFormatter() throws Exception {
         File snapDir = new File(testData, "invalidsnap");
         File logfile = new File(new File(snapDir, "version-2"), "log.274");
         String[] args = {logfile.getCanonicalFile().toString()};
-        LogFormatter.main(args);
+        org.apache.zookeeper.server.LogFormatter.main(args);
```

Comment:
Why have you removed the import and using full class name instead?

Reply:
because you cannot but @SuppressWarnings("deprecated") on an "import" clause